osis-core/src/main/java/com/scality/osis/utils/ScalityModelConverter.java

```diff
@@ -203,6 +213,55 @@ private static String toAdminPolicyDescription(String accountID) {
         return DEFAULT_ADMIN_POLICY_DESCRIPTION.replace(ACCOUNT_ID_REGEX, accountID);
     }
 
+    public static CreateUserRequest toCreateUserRequest(OsisUser osisUser) {
+        OsisUser.RoleEnum role = (osisUser.getRole() != null) ? osisUser.getRole() :OsisUser.RoleEnum.UNKNOWN ;
+        return new CreateUserRequest()
+                .withUserName(osisUser.getCdUserId())
+                .withPath(
+                        toUserPath(osisUser.getUsername(), role,
+                                osisUser.getEmail(), osisUser.getCdTenantId()));
+    }
+
+    private static String toUserPath(String username, OsisUser.RoleEnum role, String email, String cdTenantId) {
+        return "/" + username + "/" + role.getValue() + "/" + email + "/" + cdTenantId + "/" ;
+    }
```

Comment:
would you be able to add a review commit to the design doc?
Once we have the commit there, we can ask people who have reviewed already to review just that commit, and then we can merge this.

Reply:
Here is the commit for the recent design changes:
c1cc56602b590be96bfda10223fd7da94132b909

Reply:
Merging this branch. Thanks